spring-kafka/src/test/java/org/springframework/kafka/kstream/KafkaStreamsTests.java

```diff
@@ -80,8 +80,8 @@
 	@Autowired
 	private SettableListenableFuture<String> resultFuture;
 
-	@Autowired
-	private KafkaEmbedded kafkaEmbedded;
+//	@Autowired
+//	private KafkaEmbedded kafkaEmbedded;
```

Comment:
The field is not used in the test (STS `unused` warning - and you know I hate those warnings 😄 )

We can remove it altogether; I left it as a comment so as to draw your attention during the review !